charts/gardener/gardenlet/templates/clusterrole-gardenlet.yaml

```diff
@@ -162,6 +162,7 @@ rules:
   - apps
   resources:
   - deployments
+  - daemonsets
```

Comment:
What is the reason for this change? I assume gardenlet only needs permissions for DaemonSets in the garden namespace and not in any namespace. I suggest moving this back to the Role.

Reply:
Fluent Operator has ClsuterRole with this Rule already:
```
				{
					APIGroups: []string{"apps"},
					Resources: []string{"daemonsets", "statefulsets"},
					Verbs:     []string{"create", "delete", "get", "list", "patch", "update", "watch"},
				},
```